docs/book/src/cronjob-tutorial/gvks.md

```diff
@@ -54,7 +54,7 @@ The goal of this command is to create Custom Resource (CR) and Custom Resource D
 
 ## But, why create APIs at all?
 
-New APIs are how we teach Kubernetes about our custom objects. The Go structs are used to generate a Custom Resource Definition (CRD) which includes the schema for our data as well as tracking data like what our new type is called. We can then create instances of our custom objects which will be managed by our [controllers][controllers].
+New APIs are how we teach Kubernetes about our custom objects. The Go structs are used to generate a CRD which includes the schema for our data as well as tracking data like what our new type is called. We can then create instances of our custom objects which will be managed by our [controllers][controllers].
```

Comment:
WYT about we keep `Custom Resource Definition (CRD)` and link the k8s docs here: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions

Reply:
That is already being done on line 53 (not visible in the PR view, you would have to look at the whole file), so I think it would be redundant here.

Reply:
Catcher so that shows fine.